build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import com.example.buildsrc.MyExtension
+
+plugins {
+    id("java")
+}
+
+group = "org.example"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.1")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.1")
+}
+
+tasks.getByName<Test>("test") {
+    useJUnitPlatform()
+}
+
+allprojects{
+    extensions.create("Extension", MyExtension::class.java)
+
+    //passing a no-argument lambda makes the val initialized with lambda result
```

Comment:
This comment probably applies to `addElementFakeLambdaAllScope` right?

Reply:
Yes, that's my mistake, the comment should be lower.